style.css

```diff
@@ -0,0 +1,786 @@
+    height: 50px;
+    width: 200px;
+    border-style: none;
+  }
+
+  .artists {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .artists h2 {
+    margin-bottom: 5px;
+  }
+
+  .tiny-line {
+    background-color: #ec5242;
+    width: 100px;
+    height: 2px;
+    margin-bottom: 20px;
+  }
+
+  .speaker {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .skrillex {
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .skrillex img {
+    padding-top: 20px;
+    width: 100%;
+    border-radius: 50%;
+  }
+
+  .marshmello {
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .marshmello img {
+    width: 100%;
+    border-radius: 45%;
+  }
+
+  .ice {
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .ice img {
+    padding-left: 15px;
+    border-radius: 100%;
+    width: 80%;
+    height: 100%;
+  }
+
+  .eminem {
+    margin-top: 15px;
+    width: 150px;
+    height: 150px;
+    margin-right: 15px;
+  }
+
+  .eminem img {
+    padding-left: 15px;
+    border-radius: 100%;
+    width: 80%;
+    height: 100%;
+  }
+
+  .speaker-text {
+    width: 50%;
+  }
+
+  .speaker-text h1 {
+    font-family: "LatoWebBlack", sans-serif;
+  }
+
+  .job {
+    font-family: "LatoWeb", sans-serif;
+    font-style: italic;
+    color: #ec5242;
+  }
+
+  .small-grey-line {
+    width: 40px;
+    height: 2px;
+    background: #9e9e9e63;
+  }
+
+  .description {
+    font-family: "LatoWebThin", sans-serif;
+    font-size: medium;
+    font-weight: bold;
+  }
+
+  .button-artists {
+    display: flex;
+    background-color: white;
+    margin-top: 20px;
+    width: 90%;
+    height: 50px;
+    margin-bottom: 20px;
+    border-style: none;
+    border: 2px solid #9e9e9e63;
+    font-family: "LatoWebBold", sans-serif;
+    font-size: medium;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .button-artists img {
+    width: 25px;
+    padding-bottom: 4px;
+  }
+
+  .upperpart {
+    display: none;
+  }
+
+  .header-p-desktop {
+    display: none;
+  }
+
+  .header-day-desktop {
+    display: none;
+  }
+
+  .header-place-desktop {
+    display: none;
+  }
+
+  .footer-desktop {
+    display: none;
+  }
+
+  .link-desktop {
+    display: none;
+  }
+
+  .grid-container-desktop {
+    display: none;
+  }
+
+  .rights-desktop {
+    display: none;
+  }
+
+  .logo-desktop {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  #stickers {
+    display: none;
+  }
+
+  .upperpart {
+    display: flex;
+    flex-direction: row;
+    background-color: #272a31;
+    height: min-content;
+    align-items: center;
+    place-content: flex-end;
+    padding-right: 15%;
+  }
+
+  .upperpart img {
+    width: 30px;
+    padding-right: 20px;
+  }
+
+  .upperpart h4 {
+    color: white;
+    font-family: "LatoWebThin", sans-serif;
+    font-size: small;
+    margin-right: 25px;
+  }
+
+  .artists {
+    display: block;
+  }
```

Comment:
- [ ] Once again I advise you to stick to the original design when implementing the layout of the "Featured speakers" section as it should be a 2-column layout with 3 speakers on each column.
- Behance design:
  ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/67627fa7-e378-4c2d-8bd1-056c3770bf0e)
- Your design:
  ![image](https://github.com/SaveryIV/First-Capstone-Project/assets/104968236/230a9cd7-8fd3-4248-bd1d-aad7b8c07e69)